src/handbook/marketing/website.md

```diff
@@ -10,6 +10,16 @@ navTitle: Marketing - Website
 - When mentioning [FlowFuse Concepts](/docs/user/concepts/) (terminology) where possible we should link to an explanation of that concept.
 - All written content should use the [Oxford Comma](https://en.wikipedia.org/wiki/Serial_comma). We believe the Oxford Comma reduces the ambiguity of written technical content.
 
+### Website Inspiration
+
+For the creative aspect of the website and its UX, we use our [Refero](https://refero.design/) account for inspiration. Additionally, we refer to a few B2B sites for various page patterns—whether it's the layout, user flow, or interactions—, and this may extend to other aspects as well. Here's a list of our top 5:
```

Comment:
```suggestion
For the creative aspect of the website and its UX, we refer to a few B2B sites for various page patterns whether it's the layout, user flow, or interactions, and this may extend to other aspects as well. Here's a list:
```